Formula/opentsdb.rb

```diff
@@ -0,0 +1,151 @@
+class HbaseLZORequirement < Requirement
+  fatal true
+
+  satisfy(:build_env => false) { !Tab.for_name("hbase").without?("lzo") }
+
+  def message; <<-EOS.undent
+  hbase must not have disabled lzo compression to use it in opentsdb:
+    brew install hbase
+    not
+    brew install hbase --without-lzo
+  EOS
+  end
```

Comment:
The indentation is slightly off here. We prefer it to be as follows (see also most `caveats` methods of other formulae):

```ruby
def message; <<-EOS.undent
  hbase must not have disabled lzo compression to use it in opentsdb:
    brew install hbase
    not
    brew install hbase --without-lzo
  EOS
end
```